server/routes/journeys/develop-an-initial-plan/intervention-details/controller.ts

```diff
@@ -0,0 +1,63 @@
+import { Request, Response } from 'express'
+import { SchemaType } from './schemas'
+import { formatInputDate } from '../../../../utils/datetimeUtils'
+import { parseIndex } from '../subJourneyUtils'
+import { IdentifiedNeed } from '../../../../@types/express'
+
+export class InterventionDetailsController {
+  GET = async (req: Request, res: Response) => {
+    const { success, isNew, index } = parseIndex(req)
+
+    if (!success) {
+      return res.status(404).redirect('/pages/404')
+    }
+
+    return res.render('develop-an-initial-plan/intervention-details/view', {
+      identifiedNeeds: this.getData(req, res, 'identifiedNeed', isNew, index),
+      targetDate: this.getData<string | undefined>(req, res, 'targetDate', isNew, index, formatInputDate),
+      responsiblePerson: this.getData(req, res, 'responsiblePerson', isNew, index),
+      intervention: this.getData(req, res, 'intervention', isNew, index),
+      backUrl: 'summarise-identified-need',
+    })
```

Comment:
the current url for this page is `/intervention-details/{index}`, hence the back link should go to `../summarise-identified-need/{index}`.
and that's only for adding new identified need.
if it's editing a previously saved identified need, the back link should goes to `../identified-needs` (mini CYA page), refer to: https://docs.google.com/spreadsheets/d/1X-wgRPQxc-dKVrWOFC7A4T-8cEcN8ZXscjFKk1Oonbs/edit?gid=250033754#gid=250033754